lib/Chat/Changelog/Manager.php

```diff
@@ -130,6 +130,10 @@ public function getChangelogs(): array {
 			. $this->l->t('- Messages can now be edited by logged-in authors and moderators for 6 hours') . "\n"
 			. $this->l->t('- Unsent message drafts are now saved in your browser ') . "\n"
 			. $this->l->t('- *Preview:* Text chatting can now be done in a federated way with other Talk servers'),
+			$this->l->t('## New in Talk %s', ['20']) . "\n"
+			. $this->l->t('- Moderators can now ban accounts and guests to prevent them from rejoining a conversation') . "\n"
+			. $this->l->t('- Show upcoming calls when linked in events and out-of-office replacements') . "\n"
+			. $this->l->t('- *Preview:* Calls can now be done in a federated way with other Talk servers (Requires the high-performance signaling backend)'),
```

Comment:
`requires the external signaling server` was used in previous changelog (for recording consent)

Reply:
I'll stick to `High-performance backend` now as that is how the admin section is called :shrug: 
But we can adjust the old one